src/System.Collections.NonGeneric/tests/System.Collections.NonGeneric.Tests.csproj

```diff
@@ -8,6 +8,8 @@
     <RootNamespace>System.Collections.NonGeneric.Tests</RootNamespace>
     <AssemblyName>System.Collections.NonGeneric.Tests</AssemblyName>
     <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
+    <TargetFrameworkVersion>v5.0</TargetFrameworkVersion>
+    <TargetFrameworkProfile />
```

Comment:
ResolveAssemblyReferences gives this warning otherwise (and no reference to the assembly is allowed):
````
warning MSB3274: The primary reference "xunit.performance.core" could not be resolved because it was built against the ".NETPortable,Version=v5.0" framework. This is a higher version than the currently targeted framework ".NETPortable,Version=v4.5,Profile=Profile7".
````
The performance assemblies are built like so:
https://github.com/Microsoft/xunit-performance/blob/master/src/xunit.performance.core/xunit.performance.core.csproj#L18